forex-service/src/main/java/lk/ac/ucsc/forexservice/ExchangeValue.java

```diff
@@ -0,0 +1,77 @@
+package lk.ac.ucsc.forexservice;
+
+import javax.persistence.Id;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "Exchange_Value")
+public class ExchangeValue {
+
+    @Id
+    private Long id;
+
+    @Column(name = "currency_from")
+    private String from;
+
+    @Column(name = "currency_to")
+    private String to;
+
+    private double conversionRate;
+    private int port;
+
+    public ExchangeValue() {
```

Comment:
@Binurim do u need the empty constructr?

Reply:
Yes

Reply:
If deleted its not working. need to find the reason. Will do it